.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,61 @@
-<!--
-⚠️ 🚨 ⚠️  STOP AND READ THIS ⚠️ 🚨 ⚠️
+### Fixes [MC-](https://cloud-ops.atlassian.net/browse/MC-)
 
-👆👆 see that 'base fork' dropdown above? You should change it! The default value of "lord/slate" submits your change to ALL USERS OF SLATE, not just your company. This is PROBABLY NOT WHAT YOU WANT.
+#### Changes made
+- ...
+
+#### Related PRs
+- []()
+
+<!-- CLOUDMC-API-DOCS TEMPLATE
+
+### Upgrade release
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/k8s/my-env/releases/my-release/aerospike?operation=upgrade"
+```
+> Request body example(s):
+
+```js
+// Add request descriptions (use 'js' instead of 'json', else comments appear in red)
+// Change to the latest version of a chart
+{
+  "upgradeChart":  "stable/aerospike",
+  "upgradeChart":  1 
+}
+```
+
+```js
+// Change to a specific version of a chart
+{
+  "upgradeChart" : "https://kubernetes-charts.storage.googleapis.com/aerospike-0.3.2.tgz"
+}
+```
+> The above command(s) return(s) JSON structured like this:
+
+```js
+{
+  "taskId": "c50390c7-9d5b-4af4-a2da-e2a2678a83e8",
+  "taskStatus": "SUCCESS"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/releases/:id?operation=upgrade</code>
+
+Upgrade a release in a given [environment](#administration-environments)
```

Comment:
no period on this sentence?

Reply:
FIxed [here](https://github.com/cloudops/cloudmc-api-docs/pull/134/commits/996dc0a649c40dabc549d50add9dc3ba75bebec9) and added periods to all section descriptions